bskyogcard/src/components/Text.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import {atoms as a, style as s, theme as t} from '../theme/index.js'
+
+export function Text({
+  children,
+  cx,
+}: {
+  children?: React.ReactNode
+  cx?: Record<string, any>[]
+}) {
+  return (
+    <div
+      style={s([
+        a.flex,
+        a.flex_wrap,
+        {
+          columnGap: '1px',
+        },
```

Comment:
It would appear that this is the best way to wrap text correctly https://github.com/vercel/satori/issues/484